app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+  def new
+    @user = current_user
+    @post_id = params.require('id')
```

Comment:
[REQUIRED] Kindly create a comment instance here instead like this `@comment = Comment.new`